lab-edward-smith/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "lab-edward-smith",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "fs-readfile-promise": "^3.0.0",
```

Comment:
Looks like you're not using this. Be sure to get rid of any unnecessary dependencies.